test/externalTests/common.sh

```diff
@@ -232,11 +235,15 @@ function run_test
 
 function truffle_run_test
 {
-    local compile_fn="$1"
-    local test_fn="$2"
+    local soljson="$1"
+    local compile_fn="$2"
+    local test_fn="$3"
+    local force_abi_v2_flag="$4"
+
+    test [ "$force_abi_v2_flag" = force_abi_v2 ] || [ "$force_abi_v2_flag" = no_force_abi_v2 ]
```

Comment:
More non shellcheck related changes here

Reply:
Similar to https://github.com/ethereum/solidity/pull/10464#discussion_r534117440. Shellcheck complained about `FORCE_ABIv2` being an uninitialized global.

Reply:
Since `force_abi_v2` is also the name of a function, could you please rename it for readability?

Reply:
Also, I'm not sure what the `test` achieves. Is it meant to silence a shellcheck warning by making use of something that was previously initialised?

Reply:
Good point. I'll rename `force_abi_v2`. I was struggling to find something better but maybe just using something like `force-abi-v2` will make this more apparent.

The test is just an assertion/validation. It ensures that you'll get an error (due to `set -e` at the top of the file) if you try to pass anything other than `force_abi_v2` or `no_force_abi_v2` in the `force_abi_v2_flag` parameter. Since I'm changing the signature of the function and Bash is very lax about parameters, I wanted to have an additional sanity check in case I messed this up or forgot to convert one of the calls.

Reply:
Renamed to `FORCE-ABI-V2`

Reply:
Last question before approving: None of the external tests force ABIv2. If so, do we need a separate flag?

Reply:
Clarified on gitter. tl;dr this is legacy behavior which has simply been refactored.